web_listview_invert_selection/static/src/js/web_listview_invert_selection.js

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2017 Onestein
+* License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+
+odoo.define("web_listview_invert_selection", function (require) {
+    "use strict";
+    var core = require('web.core');
+    var _t = core._t;
+    var ListRenderer = require('web.ListRenderer');
+
+    ListRenderer.include(/** @lends instance.web.ListView# */{
+
+        events: _.extend({}, ListRenderer.prototype.events, {
+            'click .o_invert_selection': '_onInvertSelection',
+        }),
+
+        _onInvertSelection: function (event) {
+            event.stopPropagation();
+            var checked = self.$("tbody .o_list_record_selector input:checked");
```

Comment:
`self` doesn't seem to be defined. Is it still needed? The functionality works in runbot :confused: